src/main/java/racing/domain/car/Car.java

```diff
@@ -0,0 +1,62 @@
+package racing.domain.car;
+
+import java.util.Objects;
+
+public class Car {
+
+    private static final int STANDARD_NUMBER_MOVE_FORWARD = 4;
+    private static final int INITIATE_POSITION_VALUE = 0;
+
+    private final Name name;
+    private Position position;
+
+    public Car(String name) {
+        this(name, INITIATE_POSITION_VALUE);
+    }
+
+    public Car(String name, int value) {
+        this.name = new Name(name);
+        this.position = new Position(value);
+    }
```

Comment:
메서드 오버로딩의 경우 되도록 `생성자 메서드 체이닝`을 활용해주세요! 🤗
```suggestion
    public Car(String name, int position) {
        this(name, new Position(position);
    }
```

Reply:
넵! 리뷰 감사합니다! 코드에 반영했습니다! 😄